src/aws_scanner_config.py

```diff
@@ -175,11 +179,13 @@ def logs_route53_log_group_delivery_role_assume_policy(self) -> Dict[str, Any]:
     def logs_vpc_log_group_delivery_role_policy_document(self) -> Dict[str, Any]:
         return self._get_json_config("logs", "vpc_log_group_delivery_role_policy_document")
 
-    def logs_vpc_log_group_retention_policy_days(self) -> int:
-        return self._get_int_config("logs", "vpc_log_group_retention_policy_days")
-
-    def logs_route53_log_group_retention_policy_days(self) -> int:
-        return self._get_int_config("logs", "route53_log_group_retention_policy_days")
+    def logs_group_retention_policy_days(self, service_name: ServiceName) -> int:
+        init_config = 0
+        if service_name == ServiceName.vpc:
+            init_config = self._get_int_config("logs", "vpc_log_group_retention_policy_days")
+        elif service_name == ServiceName.route53:
+            init_config = self._get_int_config("logs", "route53_log_group_retention_policy_days")
```

Comment:
same comment here about keeping logic out of config

Reply:
Invalid name handled by raising an exception

Reply:
wrt keeping logic out of config - we believe that passing the service name to the generic function provides more flexibility to read as many configs as needed for the specific service.